Sources/Rendering/OpenGL/Texture/index.js

```diff
@@ -1222,19 +1222,103 @@ function vtkOpenGLTexture(publicAPI, model) {
         halfFloatExt && model.openGLDataType === halfFloatExt.HALF_FLOAT_OES;
     }
 
-    if (!useHalfFloat) {
-      return false;
+    if (
+      useHalfFloat &&
+      (hasExactHalfFloat(offset, scale) || preferSizeOverAccuracy)
+    ) {
+      model.useHalfFloat = true;
+    } else {
+      model.useHalfFloat = false;
     }
+  }
 
-    // Don't consider halfFloat and convert back to Float when the range of data does not generate an accurate halfFloat
-    // AND it is not preferable to have a smaller texture than an exact texture.
```

Comment:
why getting rid of this comment ?

Reply:
Don't know why or how, will put them back